include/OpenSpaceToolkit/Physics/Environment/Atmospheric/Earth/Exponential.hpp

```diff
@@ -38,7 +38,7 @@ using ostk::physics::environment::atmospheric::Model;
 
 /// @brief                      Exponential atmospheric model
 
-class Exponential : public Model
```

Comment:
Is this desired? I thought the architecture was to have all of these inherit from an Atmospheric model base object?

Reply:
Yeah, this is undoing something that was a huge pain when I first implemented these. 
The problem: we want a super-class for atmosphere models (`Model.hpp`) so that we are able to make atmospheric models for planets that are _not Earth_ and handle them abstractly in the Celestial Object. 

When I implemented these at first, I thought to myself "hey, Exponential and NRLMSISE are the actual models, they should _also_ inherit from this class."

In practice it just means they must have a method `getDensity(Position, Instant)`. At this point I think it makes more sense to just consider these as wrappers that cover the underlying math. And since that math operates on LLA, it doesn't make sense for them to inherit from this `Model.hpp`.

The Earth atmospheric model class `Atmospheric/Earth.hpp` still inherits from `Model`

Reply:
Ah gotcha I understand. That makes sense.